buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -418,10 +418,40 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Build and run tests",
+      "displayName": "Test Msi",
       "timeoutInMinutes": 0,
       "condition": "and(succeeded(), ne(variables.PB_SkipTests, 'true'))",
       "refName": "Task19",
+      "task": {
+      "id": "c6c4c611-aa2e-4a33-b606-5eaba2196824",
+      "versionSpec": "1.*",
+      "definitionType": "task"
+    },
+    "inputs": {
+      "solution": "$(PB_SourcesDirectory)\\src\\test\\dir.proj",
```

Comment:
I'd rather see us add the tests to Jenkins rather than the official build.

Our official builds take long enough the way it is.

Reply:
And also - if these tests are going to break, they should break on PR.  Not post-merge of the PR.

Reply:
It is probably worth adding the test to both jenkins and official builds.